libnavui-dropin/api/current.txt

```diff
@@ -0,0 +1,563 @@
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? infoPanelTripProgressBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? maneuverBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? roadNameBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? speedLimitBinder;
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class ViewOptionsCustomization {
+    ctor public ViewOptionsCustomization();
+    method public String? getMapStyleUriDay();
+    method public String? getMapStyleUriNight();
+    method public com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? getRouteArrowOptions();
+    method public com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? getRouteLineOptions();
+    method public void setMapStyleUriDay(String? p);
+    method public void setMapStyleUriNight(String? p);
+    method public void setRouteArrowOptions(com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? p);
+    method public void setRouteLineOptions(com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? p);
+    property public final String? mapStyleUriDay;
+    property public final String? mapStyleUriNight;
+    property public final com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? routeArrowOptions;
+    property public final com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? routeLineOptions;
+    field public static final com.mapbox.navigation.dropin.ViewOptionsCustomization.Companion Companion;
+  }
+
+  public static final class ViewOptionsCustomization.Companion {
+    method public com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions defaultRouteArrowOptions(android.content.Context context);
+    method public com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions defaultRouteLineOptions(android.content.Context context);
+  }
+
+}
+
+package com.mapbox.navigation.dropin.binder {
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public interface Binder<T> {
+    method public com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver bind(T? viewGroup);
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public fun interface UIBinder extends com.mapbox.navigation.dropin.binder.Binder<android.view.ViewGroup> {
+    field public static final com.mapbox.navigation.dropin.binder.UIBinder.Companion Companion;
+  }
+
+  public static final class UIBinder.Companion {
+    method public com.mapbox.navigation.dropin.binder.UIBinder getUSE_DEFAULT();
+    property public final com.mapbox.navigation.dropin.binder.UIBinder USE_DEFAULT;
+  }
+
+  public final class UIBinderKt {
+  }
+
+}
+
+package com.mapbox.navigation.dropin.component.audioguidance {
+
+  public abstract sealed class AudioAction {
```

Comment:
Can this be converted to `internal`?

Reply:
Let's not make any actions internal for now as they are the only means to communicate from outside. We can revisit this decision once we figure out how to expose public API(s) based on state centralization. We can make the actions Experimental though.